STYLEGUIDE.md

```diff
@@ -0,0 +1,1336 @@
+# Java Style Guide
+
+###### This is a modified version of the excellent [Twitter Java Style Guide](https://github.com/twitter/commons/blob/master/src/java/com/twitter/common/styleguide.md).
+
```

Comment:
@jakubcech please ascertain that this notice suffices to comply with the license restrictions.

Reply:
There's no **notice** that would have to be retained. We conform to the requirement that we must state that we've modified the file. And I don't think there's a need to list any significant changes in this case either. lgtm